meta-genivi-dev/meta-genivi-dev/recipes-dev-platform/images/genivi-dev-platform.bb

```diff
@@ -32,6 +32,7 @@ IMAGE_INSTALL_append = " \
     packagegroup-nodejs-runtime \
     packagegroup-gdp-qt5 \
     packagegroup-gdp-rvi \
+    packagegroup-gdp-vsi \
```

Comment:
I think we might have a discussion/redesign regarding package groups.  There's a risk of having almost one per component, which isn't the purpose. 

The group you created here could serve as the "vehicle-signaling-subsystem" group for any related components.

We might rather call it -vss  although to be clear vss stands for v.s.specification, not v.s.subsystem.  Nonetheless, vss might be a useful name for this group.